test/integration/acl-tls-test.js

```diff
@@ -50,22 +53,32 @@ var userCredentials = {
   }
 }
 
-describe('ACL with WebID+TLS', function () {
+// TODO Remove skip. TLS is currently broken, but is not a priority to fix
```

Comment:
maybe add:
... since the current Solid spec does not require supporting webid-tls on the resource server. The current spec only requires the resource server to support webid-oidc, and it requires the IDP to support webid-tls as a log in method, so that users of a webid-tls client certificate can still use their certificate (and not a username/password pair or other login method) to "bridge" from webid-tls to webid-oidc.

Reply:
or something to that extent. Would be even better if you link to a spec issue where this is stated, will post here if i find one.

Reply:
ah, found it: https://github.com/solid/solid-spec/pull/171#issuecomment-506350514